src/components/Advantages/index.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Text from "../Text";
+import Icon from "../Icon";
+import classnames from "classnames";
+import classNames from "classnames";
+
+const Advantages = ({ colorSchema, content }) => {
+  const colorTextClass =
+    colorSchema === "dark"
+      ? "py-11 px-10 gap-20 bg-light-on-background-900 text-light-on-background-50"
+      : "text-gray-900 gap-10";
+
+  const colorIconsClass =
+    colorSchema === "dark" ? "bg-light-on-background-800" : "bg-light-on-background-900";
+
+  return (
+    <div className={classnames("grid grid-cols-3 ", colorTextClass)}>
```

Comment:
This grid should be responsive. In other words, 3 columns for desktop but for mobile, the blocks should go one by one vertically. I think there is a way to do that with Tailwind. 
This is an article on the topic https://daily-dev-tips.com/posts/tailwind-grid-responsive-4-column-blocks/

Reply:
UPD: same thing with the grid in the `Card` component (in the stories). Perhaps, it has never been responsive but I believe that has to be fixed.